src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/DeprecatedListWriter.java

```diff
@@ -81,19 +80,22 @@ public static void generate(HtmlConfiguration configuration) throws DocFileIOExc
     @Override
     protected void addExtraSection(DeprecatedAPIListBuilder list, Content content) {
         List<String> releases = configuration.deprecatedAPIListBuilder.releases;
-        if (!releases.isEmpty()) {
+        if (releases.size() > 1) {
             Content tabs = HtmlTree.DIV(HtmlStyle.checkboxes, contents.getContent(
                     "doclet.Deprecated_API_Checkbox_Label"));
             for (int i = 0; i < releases.size(); i++) {
-                int releaseIndex = i + 1;
                 String release = releases.get(i);
+                String releaseIndex = release.isEmpty() ? "" : Integer.toString(i + 1);
+                Content releaseLabel = release.isEmpty()
+                        ? contents.getContent("doclet.Deprecated_API_Checkbox_Other_Releases")
+                        : Text.of(release);
                 HtmlId htmlId = HtmlId.of("release-" + releaseIndex);
```

Comment:
I chose the empty string to represent other/uncategorized releases because we can't make any assumptions about release naming except that release names are non-empty strings. However I recognize that the code above is not great in terms of naming and comments. I have added a subsequent commit in which I pulled out the checkbox generating code into a separate method, added an explanatory comment and only use the empty string where necessary (i.e. not in the HtmlId).